canonical-web-and-design/ubuntu.com
# Add redirect from /IP to the legal page

## Done

- Add redirect from /IP to /legal/intellectual-property-policy to use in legal documents

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/IP and see that it goes to http://0.0.0.0:8001/legal/intellectual-property-policy

## Issue / Card

Fixes #5485
